README.md

```diff
@@ -60,16 +58,16 @@ testing.
 ### Caveats
 
 * The Lambda is never told the port that the API Gateway is listening
-  on. Most APIs will listen on `443` (HTTPS), so that's what's in this
-  example.
+  on. Most APIs will listen on `443` (HTTPS), so that's what's in the
+  above example.
```

Comment:
```suggestion
  on. Most APIs will listen on `443` (HTTPS), so that's what the
  library reports by default. See `runWithContext` if you need to
  change this.
```